### PR TITLE
Declare all the python3 binaries properly + venv installed automatically.

### PR DESCRIPTION
**Pull request by jcpassy**

### CODE REVIEW

File: tasks/install_python3_ubuntu.yml [hunk 1]
```diff
@@ -12,25 +12,30 @@
...
+      bamboo_capabilities: "{{ (bamboo_capabilities | default({})) | combine({item.stdout_lines.0:item.stdout_lines.1}) }}"
+    loop: "{{ python3_output_ubuntu.results }}"
     tags:
       - capability
 
   when: (ansible_distribution=="Ubuntu")
-
   tags:
-    - python3
+    - python3
```
Comment (raffienficiaud) [inline tasks/install_python3_ubuntu.yml]: line break at end file

File: tasks/install_python3_ubuntu.yml [hunk 2]
```diff
@@ -12,25 +12,30 @@
...
 
-  - name: '[PYTHON3] locating python3 binary'
-    shell: python3 -c "import sys; print('%d.%d\n%s' % (sys.version_info.major, sys.version_info.minor, sys.executable))"
+  - name: '[PYTHON3] locating python3 binaries'
+    shell: "{{ item.key }} -c \"{{ item.value }}\""
     register: python3_output_ubuntu
     ignore_errors: False
+    loop:
+      - key: python3
+        value: import sys; print('system.builder.command.python%d\n%s' % (sys.version_info.major, sys.executable))
```
Comment (raffienficiaud) [inline tasks/install_python3_ubuntu.yml]: would it be possible to pass only the part that is specific to each of the keys instead ? (such as `python%d.%d`)

**Comment by raffienficiaud** [comment]
Thanks! minor comments

File: tasks/install_python3_ubuntu.yml (hunk 1 above)
Comment (jcpassy) [reply tasks/install_python3_ubuntu.yml]: Done!

File: tasks/install_python3_ubuntu.yml (hunk 2 above)
Comment (jcpassy) [reply tasks/install_python3_ubuntu.yml]: Fixed as well!

**Comment by raffienficiaud** [comment]
Thanks!

**Pull request merged**